subprojects/workers/src/integTest/groovy/org/gradle/workers/internal/WorkerDaemonFailureLoggingIntegrationTest.groovy

```diff
@@ -41,12 +42,14 @@ class WorkerDaemonFailureLoggingIntegrationTest extends AbstractDaemonWorkerExec
         fails("runInWorker")
 
         and:
-        def taskOperation = buildOperations.first(ExecuteTaskBuildOperationType) {
-            it.displayName == "Task :runInWorker"
+        def taskActionOperation = buildOperations.first(ExecuteTaskActionBuildOperationType) {
+            it.displayName == "Execute executeTask for :runInWorker"
```

Comment:
Same here.  Let's make sure to update the method name so that it's clear the expectation is now that the message _will_ be associated with the task.